### PR TITLE
core(large-javascript-libraries): delete

### LINKED ISSUE #12296: large-javascript-libraries rework proposal
We moved `large-javascript-libraries` out of the default config after receiving some negative feedback from developers concerned about Google choosing winners based on an unclear criteria. I collected some potential action items to [fix this audit incrementally](https://github.com/GoogleChrome/lighthouse/issues/11423), but instead here I want to propose an entirely different approach.

* Keep as diagnostic audit (don't upgrade to an opportunity)
* Require source maps so we know the actual size and can detect packages without `js-library-detector`
* Show all packages larger than 200 kB (uncompressed)
* Link to https://bundlephobia.com/result?p=<pkgname>
* Change title to `Evaluate if large libraries on this page are necessary or have smaller alternatives`

EDIT: TIL https://packagephobia.com/result?p=lighthouse

### PR DESCRIPTION
**Pull request by connorjclark**
fixes #12296

### CODE REVIEW

**Review by adamraine — APPROVED** [review]
I was expecting all our locales to be updated, will that only happen when we do a i18n import?

**Comment by connorjclark** [comment]
yeah

File: package.json [hunk 1]
```diff
@@ -129,7 +129,6 @@
     "archiver": "^3.0.0",
     "browserify": "^17.0.0",
     "browserify-banner": "^1.0.15",
-    "bundle-phobia-cli": "^0.14.6",
```
Comment (brendankenny) [inline package.json:132]: shouldn't there be a `yarn.lock` change for this? And if there is, why doesn't `git add -A && git diff --cached --exit-code` catch it?

Comment (brendankenny) [reply package.json:132]: ohhh, `yarn install --frozen-lockfile` leaves `yarn.lock` unchanged. lol

Comment (connorjclark) [reply package.json:132]: oops. let's remove that

Comment (brendankenny) [reply package.json:132]: Interesting, `--fronzen-lockfile` seems to only have an error exit code when it needs to install new packages, not uninstall. It seems like any change to the `yarn.lock` file should trigger that, but I guess at least it's just unused modules sitting there, not missing ones?

Comment (brendankenny) [reply package.json:132]: https://github.com/yarnpkg/yarn/issues/5840#issuecomment-415209523

> The first issue is the situation described by you, where `--frozen-lockfile` succeeds even if there are optimizations to be made (i.e. packages to remove). This behavior is intentional, though there are good arguments to be made that this behavior should be changed. The documentation should definitely be improved here (https://github.com/yarnpkg/yarn/issues/5847), but I don't think it's a bug.

much later...

> Just for clarity - was this ever fixed for Classic Yarn (1.x)? Facing this issue in 1.22.5 and I'd rather not have to migrate to v2.

> 1.x is frozen and will only receive security fixes, so the answer is no.

😐

**Comment by brendankenny** [comment]
`yarn.lock 358 changes: 19 additions & 339 deletions`

so nice

**Pull request merged**